doc/classes/ResourceFormatLoader.xml

```diff
@@ -99,10 +99,19 @@
 	</methods>
 	<constants>
 		<constant name="CACHE_MODE_IGNORE" value="0" enum="CacheMode">
+			Resource and subresources are neither retrieved from nor stored into the resource cache. External resources are loaded with [constant CACHE_MODE_REUSE].
 		</constant>
 		<constant name="CACHE_MODE_REUSE" value="1" enum="CacheMode">
+			Resource, subresources, and external resources are retrieved from cache if present, instead of loaded. If not present, they are loaded and stored into the cache. The same rules are propagated to external resources.
 		</constant>
 		<constant name="CACHE_MODE_REPLACE" value="2" enum="CacheMode">
+			Like [constant CACHE_MODE_REUSE], but both the main resource and its subresources, if already in the cache, have their data refreshed instead of being recreated as completely new instances.
+		</constant>
+		<constant name="CACHE_MODE_IGNORE_DEEP" value="3" enum="CacheMode">
+			Like [constant CACHE_MODE_IGNORE], but propagates recursively among loads of external resources.
```

Comment:
By external resources you mean dependencies? The description isn't very clear on that.

Reply:
Yes.

I inherited the jargon from the description in the comments and now I see it's not as clear as we may want. I'm amending the wording in the next push.